user_interaction.py

```diff
@@ -1,110 +1,108 @@
 #!/usr/bin/env python
 import os
 
-# Determining the way the library will be used
-modeOptions = {1 : "train", 2 : "eval", 3 : "sample"}
-
-print("You want to train, evaluate or sample a model?\n\
-[1] train\n[2] evaluate\n[3] sample")
-while True:
-    mode = input()
-    if mode not in modeOptions:
-        print("Insert only 1, 2 or 3")
-        continue
-    else:
-        mode = modeOptions[mode]
-        break
-
-print("You chose to " + mode + " the model.\n")
-
-# Determining if the image will run locally or remotely
-placeOptions = {1 : "locally", 2 : "on the Google Cloud AI Platform"}
-print("You want to " + mode + " locally or on the Google Cloud AI Platform?\n\
-[1] locally\n[2] Google Cloud AI Platform")
-while True:
-    place = input()
-    if place not in [1, 2]:
-        print("Insert only 1 or 2")
-        continue
-    else:
-        break
-
-print("You chose to " + mode + " " + placeOptions[place] + "\n")
-
-# Determining path for data retrieval
-dataPath = raw_input("Please insert a path for data retrieving. This can be a local directory or a GCS Bucket:\n")
+# # Determining path for data retrieval
+data_path = raw_input("Insert a path for data retrieving. \
+This can be a local directory or a GCS Bucket:\n")
 
 # Determining path for storing model, snapshots and summaries
-storingPath = raw_input("\nPlease insert a path for storing the model, snapshots and summaries.\
- This can be a local directory or a GCS Bucket:\n")
+storing_path = raw_input("\nInsert a path for saving the model, \
+snapshots and summaries. This can be a local directory or a GCS Bucket:\n")
```

Comment:
With current setup, I think it can be only GCS Bucket. If model is saved inside the VM it gets destroyed when the job finishes.